### PR TITLE
Updated input screenshots

### PR DESCRIPTION
**Pull request by NJKode**
(Except for object input screenshots, which are undergoing changes)

https://hearty-clarinet.cloudvent.net/documentation

### CODE REVIEW

**Comment by rycoll** [comment]
Do we have a nice way to shrink our imgs down? The boolean input screenshot looks a bit blown up on my big screen :/

Screenshot 2023-05-24 at 4 17 12 PM

**Comment by NJKode** [comment]
> Do we have a nice way to shrink our imgs down? The boolean input screenshot looks a bit blown up on my big screen :/
Hmm that's a good point. The previous img got around this by just being really wide. Will investigate options.

File: articles/using-select-inputs-to-edit-your-data.mdx [hunk 1]
```diff
@@ -114,14 +120,18 @@ _inputs:
 </comp.MultiCodeBlock>
 
 ## Multichoice
+
 Segmented controls input for selecting multiple values from a set of options. Functionally equivalent to a [Multiselect](#multiselect) input.
 
 Multichoice inputs are shown for inputs configured with the type `multichoice`.
 
+<comp.DocsImage path="/documentation/static/inputs/select/multichoice.png" alt=""/>
```
Comment (rycoll) [inline articles/using-select-inputs-to-edit-your-data.mdx:128]: Consider using multichoice-with-icons.png to show the UI with nice icons?

**Review by rycoll — APPROVED** [review]
Looks good!

File: articles/using-select-inputs-to-edit-your-data.mdx (hunk 1 above)
Comment (NJKode) [reply articles/using-select-inputs-to-edit-your-data.mdx:128]: Yeah I considered it when uploading, but then we would have to change the corresponding codeblock as well. Doing so would really add a lot of complexity, and I think it's better to show a really simple example. I was tempted to write an extra example showing off the multi with icons, think it's worth it?

Comment (rycoll) [reply articles/using-select-inputs-to-edit-your-data.mdx:128]: >  we would have to change the corresponding codeblock as well. Doing so would really add a lot of complexity

Oh yep, great point. I'm content with how it is, up to you! I've approved the PR anyway.

**Pull request merged**